services/brig/deb/opt/brig/bin/fetch.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
```

Comment:
Why not node?

Reply:
Because way too many dependencies and this project didn't even have the `package.json` to begin with :) So if someone would like to run this locally they can just execute this file, instead of installing bunch of stuff.. and our backend doesn't have much Node. Translating this [file](https://github.com/wireapp/wire-emails/blob/master/bin/translate.py) to `. js` though makes more sense.. :) /cc @bennyn

Reply:
Ok, I am convinced @lipis.

Reply:
I'll delete the comments (but I saved the links for future reference) to avoid the noise.